Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -207,20 +219,30 @@ namespace Multiplayer
         {
             server_rhi = static_cast<AZ::CVarFixedString>(editorsv_rhi_override);
         }
+
+        const auto console = AZ::Interface<AZ::IConsole>::Get();
+        AZ::CVarFixedString sv_defaultPlayerSpawnAsset;
+
+        if (console->GetCvarValue("sv_defaultPlayerSpawnAsset", sv_defaultPlayerSpawnAsset) != AZ::GetValueResult::Success)
```

Comment:
Since this is inside Multiplayer gem, can't you just access the value of sv_defaultPlayerSpawnAsset directly? I believe you can extern that.

Reply:
I removed extern because externing won't pick up the value set in setreg.